src/func/Microsoft.Health.Fhir.Ingest.Host/Microsoft.Health.Fhir.Ingest.Host.csproj

```diff
@@ -13,19 +13,18 @@
     <WarningsAsErrors />
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="4.1.0" />
```

Comment:
Do we know why this was removed?  We need this version of EvenHubs.Processor or higher to ensure managed identity works.

Reply:
We do need this for Managed Identity. It needs to be added back manually because we are declaring a newer version than is required by another dependency.

Reply:
I'll add it back since it is needed for managed identity. While running in docker, this package prevents the function from reading from the event hub. Will dig into it a bit more.

Reply:
Added it back. Root cause was having two functions running at the same time from the same event hub and apparently there is differing behavior between 4.1.0 and lower versions when that occurs.